pages/settings.vue

```diff
@@ -12,7 +12,6 @@ useHydratedHead({
 const route = useRoute()
 
 const isRootPath = computedEager(() => route.name === 'settings')
-const devMode = useAppConfig().env === ('dev' || 'canary' || 'preview')
```

Comment:
Removing the env check here so the link to the Privacy page will now always show up regardless of environment

<img width="1031" alt="Screenshot 2023-11-29 at 2 39 50 PM" src="https://github.com/MozillaSocial/elk/assets/2893463/905717e5-0804-4532-a16a-738ded9dad63">